modules/nf-core/tcoffee/align/tests/main.nf.test

```diff
@@ -168,10 +177,43 @@ nextflow_process {
         then {
             assertAll(
                 { assert process.success },
-                { assert snapshot(process.out.alignment).match("alignment - lib") },
                 { assert path(process.out.lib.get(0).get(1)).getText().contains("1ahl") },
-                { assert snapshot(process.out.versions).match("versions_lib") }
+                { assert snapshot(
+                                    process.out.alignment,
+                                    process.out.versions
+                                 ).match()
+                }
+            )
+        }
+    }
+
+    test("stub") {
+
+        config "./sequence.config"
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ],
+                             file(params.modules_testdata_base_path + "../../multiplesequencealign/testdata/setoxin-ref.fa", checkIfExists: true)
+                           ]
+                input[1] = [[:],[]]
+                input[2] = [[:],[],[]]
+                input[3] = true
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                                    process.out.alignment,
+                                    process.out.versions
+                                 ).match()
```

Comment:
I think this can be aligned nicely
```suggestion
                { assert snapshot(
                         process.out.alignment,
                         process.out.versions
                         ).match()
```